### PR TITLE
Fixes #194 Modal broken Links

### LINKED ISSUE #194: Modal Links are broken.
##Modal Links are broken and takes to 404 page.

Screenshot 2022-03-02 at 12 51 03 PM

Solution :
These can be linked by 
1) Infragram filter packs : https://store.publiclab.org/collections/featured-kits/products/infragram-diy-filter-pack?variant=1058088684
2) Webcam : https://store.publiclab.org/collections/featured-kits/products/infragram-pi-camera?variant=14249592029293

As this artical i.e [Infragram Point & Shoot](https://publiclab.org/wiki/infragram-point-shoot) 

3) Infragram Point & Shoot cameras: https://store.publiclab.org/collections/featured-kits/products/infragram-pi-camera?variant=14249592029293

@jywarren and @TildaDares can I work on this.

### PR DESCRIPTION
**Pull request by Syed-Ansar**
Modal Links which are broken are fixed #194 .

Solution:

Products which are available in the Public lab store are replaced with broken links

Screenshot 2022-03-03 at 12 15 51 PM
.

### CODE REVIEW

**Comment by Syed-Ansar** [comment]
@jywarren Please review my Pull Request.

File: index.html [hunk 1]
```diff
@@ -38,7 +38,17 @@
...
+                 <p>
+                 1. These auto-load common
+                 </p> 
+                 <p>
+                  2. Analysis and
+                 </p>
+                 <p>
+                  3. Colorize settings so you don't have to manually choose them.</p>
+                 </p>
+
```
Comment (TildaDares) [inline index.html:51]: Hi @Syed-Ansar, these changes are not for this PR. Can you remove them? Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: dist/infragram.css [hunk 2]
```diff
@@ -77,11 +77,6 @@ div#colorbar-container canvas#colorbar {
     height:auto;
   }
 }
-
-.container {
-  padding-left:unset;
-}
-
```
Comment (TildaDares) [inline dist/infragram.css:84]: These changes are not for this PR. Can you remove them? Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: dist/infragram.css (hunk 2 above)
Comment (Syed-Ansar) [reply dist/infragram.css:84]: Yeah! I will do that. Thank you.

File: index.html [hunk 3]
```diff
@@ -47,13 +57,13 @@ <h3>Quick start</h3>
...
-                     <td>basic plant analysis for <a href="http://store.publiclab.org/product/infragram-diy-filter-pack">Infragram filter packs</a> and <a href="http://store.publiclab.org/product/infragram-webcam">webcams</a></td>
+                     <td>Basic plant analysis for <a href="https://store.publiclab.org/collections/featured-kits/products/infragram-pi-camera?variant=14249592029293">Infragram Pi Camera</a></td>
                      <td><a class="btn btn-primary" id="preset_ndvi_blue">Basic</a>
                          <a class="btn btn-success" id="preset_ndvi_blue_color">Colorized</a></td>
                    </tr>
                    <tr>
                      <td><b>NDVI for <span style="color:red;">RED</span> filters</b></td>
-                     <td>basic plant analysis for <a href="http://store.publiclab.org/products/infragram-point-shoot-plant-cam">Infragram Point &amp; Shoot cameras</a></td>
+                     <td>Basic plant analysis for <a href="https://store.publiclab.org/collections/featured-kits/products/infragram-diy-filter-pack?_pos=1&_sid=ff960859b&_ss=r&variant=1058088684">Infragram DIY Filter Pack</a></td>
```
[the remark this answers is not in the document]
Comment (Syed-Ansar) [reply index.html:66]: Oh! I have't concentrated on that. Thank you @TildaDares I will fix this.

**Comment by Syed-Ansar** [comment]
I will make changes in new PR

**Pull request closed without merging**